src/components/NewLocationPage/SummaryStatsBlock/DesktopSummaryStat.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import MetricSubLabel from './MetricSubLabel';
+import MetricValue from './MetricValue';
+import {
+  StatContent,
+  Row,
+  MetricLabel,
+  StyledChevron,
+} from './SummaryStatsBlock.style';
+import { metricSubLabelText, SummaryStatProps } from './utils';
+
+const DesktopSummaryStat: React.FC<SummaryStatProps> = ({
+  levelInfo,
+  formattedValue,
+  hasSubLabel,
+  metricName,
+  isMobile,
+  metric,
+}) => {
+  return (
```

Comment:
I think we could make more clear if this component is responsible to know how to render each metric or receive parameters instead. 

- If we pass `metric`, then we could calculate `formattedValue`, `metricName`, `hasSubLabel` inside the component
- If we don't pass the `metric` prop (the props will be computed by the parent component), we could receive `metricSubLabelText` as an optional prop and avoiding having that logic in this component.